pkg/network/java/testutil/inject.go

```diff
@@ -17,13 +17,32 @@ import (
 )
 
 // RunJavaVersion run class under java version
-func RunJavaVersion(t *testing.T, version string, class string) {
+func RunJavaVersion(t *testing.T, version string, class string, waitFor *regexp.Regexp) {
 	t.Helper()
+	if waitFor == nil {
+		regexp.MustCompile("loading TestAgentLoaded.agentmain.*")
```

Comment:
did you mean this?
```suggestion
		waitFor = regexp.MustCompile("loading TestAgentLoaded.agentmain.*")
```

Reply:
Yes thanks I think I break (not really) in rebase this will change in the next PR